doc-content/drools-docs/src/main/asciidoc/DMN/dmn-properties-ref.adoc

```diff
@@ -52,14 +52,26 @@ When valorized with a Java fully qualified name, this property loads a DMN profi
 org.kie.dmn.runtime.listeners.$LISTENER_NAME::
 When valorized with a Java fully qualified name, this property loads and registers a DMN Runtime Listener onto the {DECISION_ENGINE} at start time.
 You can use this property to register a DMN listener in order to be notified of several events during DMN model evaluations.
-You can also configure this property in the `kmodule.xml` file in your project.
 +
 --
 ----
 -Dorg.kie.dmn.runtime.listeners.mylistener=org.acme.MyDMNListener
 ----
 //kept removed `[source]` for this last snippet because it rendered unlike all the others in community output otherwise for some reason. as per SJR comment above.
 --
++
+You can also configure this property in the `kmodule.xml` file in your project.
```

Comment:
Just a suggestion: 
You can remove the word **also** from the sentence.
You can configure this property in the `kmodule.xml` file in your project.